scripts/quests/bastok/A_Flash_in_the_Pan.lua

```diff
@@ -64,15 +64,13 @@ quest.sections =
             onEventFinish =
             {
                 [219] = function(player, csid, option, npc)
-                    if quest:complete(player) then
-                        player:confirmTrade()
-                        GetNPCByID(bastokMarketsID.npc.AQUILLINA):setLocalVar('tradeCooldown', os.time() + 900)
+                    if player:hasCompletedQuest(quest.areaId, quest.questId) then
+                        quest.reward.fame = 15
                     end
-                end,
 
-                [218] = function(player, csid, option, npc)
                     if quest:complete(player) then
-                        player:confirmTrade(false)
+                        player:confirmTrade()
+                        GetNPCByID(bastokMarketsID.npc.AQUILLINA):setLocalVar('tradeCooldown', os.time() + 900)
```

Comment:
So I think what I was trying to fix here and messed up on is that if you attempt a trade and it doesn't take the items, we need to pass `player:confirmTrade(false)` to properly clean up the trade or else it puts the items into a weird state. I think that is still needed, just implemented properly unlike what I did.